implementations/rust/ockam/ockam_command/src/tcp/outlet/list.rs

```diff
@@ -32,14 +32,14 @@ async fn run_impl(
     rpc.request(Request::get("/node/outlet")).await?;
     let response = rpc.parse_response::<OutletList>()?;
 
-    println!("Outlet:");
```

Comment:
To make the `list` command consistent with other `list` commands, it should error if there are no items to show. [Here's an example](https://github.com/build-trust/ockam/blob/42f19587ab96a2330d7dbcf4e2647d6189d2f6a6/implementations/rust/ockam/ockam_command/src/node/list.rs#L33) of how it's handled in the `node list` command.

Reply:
Added a similar error to both inlet/outlet `list` commands in the latest commit